docs/learn/getting-started-boss-room.md

```diff
@@ -0,0 +1,126 @@
+---
+id: bossroom
+title: Getting Started with Boss Room
+---
+
+![Banner](/img/banner.png)
+# Boss Room - Multiplayer Samples Co-op built with Unity MLAPI
+
+:::important
+This project is currently experimental.
+:::
+
+Boss Room is a fully functional Multiplayer Samples Co-op made in Unity and MLAPI. It is built to serve as an educational sample that showcases certain typical gameplay patterns that are frequently featured in similar games.
+
+The intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. 
+
+## To get the project
+ - A release version can be downloaded from the [Releases](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page. 
+ - Unzip the archive file.
+ - You are now ready to add the project to Unity Hub.
+
+:::important Compatability
+- The MLAPI supports all major Unity platforms. To use the WebGL platform a custom WebGL transport based on web sockets is needed.
+- MLAPI is compatible with Unity 2020.3.0f1.
+- Make sure to include standalone support for Windows/Mac in your installation. 
+:::
+
+## To add the project with Unity Hub
+1. Open Unity Hub.
+1. Click **Add**.
+1. Select the root folder of the downloaded project.
+
+  ![adding project to unity](/img/addbossroom.gif)
+
```

Comment:
Should we add a comment saying something like the MLAPI v0.1.0 package will be installed with the Boss Room project?

Reply:
I have added this text to the note now